### PR TITLE
Overview stored data

### PR DESCRIPTION
**Pull request by FreytagDom**

### CODE REVIEW

File: components/HomeInput.js [hunk 1]
```diff
@@ -11,57 +10,55 @@ import carbohydrates from '../public/carbohydrates.png';
...
-  function handleInsulinUnit(
-    bloodsugar,
-    carbohydrates,
-    morningfactor,
-    lunchfactor,
-    eveningfactor
-  ) {
+  function handleInsulinUnit(bloodsugar, carbohydrates, daytimeFactor) {
     const targetValue = 100;
     const corretionValue = 60;
```
Comment (K-Juno) [inline components/HomeInput.js]: Typo ;)

File: components/HomeInput.js [hunk 2]
```diff
@@ -11,57 +10,55 @@ import carbohydrates from '../public/carbohydrates.png';
...
 
     const calculateUnit = (
       (bloodsugar - targetValue) / corretionValue +
```
Comment (K-Juno) [inline components/HomeInput.js]: Same typo ;)

File: components/SavedData.js [hunk 3]
```diff
@@ -1,25 +1,90 @@
...
+              <br />
+              <Insulin>
+                Verwendetes Insulin: <br /> {item.insulin}
+              </Insulin>
+              <br />
+              <Factor>
+                Insulin Faktor: <br /> {item.daytimeFactor}
+              </Factor>
+              <InsulinUnits>
+                gepritzte Insulin <br /> Menge: {item.calculateUnit}
```
Comment (K-Juno) [inline components/SavedData.js]: Typo ;)

File: pages/setSavedData.js [hunk 4]
```diff
@@ -0,0 +1,33 @@
...
+    const response = await fetch('/api/setInsulinDatas', {
+      method: 'POST',
+      headers: {
+        'content-type': 'application/json',
+      },
+
+      body: JSON.stringify(cardData),
+    });
+    const json = response.json();
+    // console.log(response_);
```
Comment (K-Juno) [inline pages/setSavedData.js]: There is a console.log / comment. ;)

File: pages/setSavedDatas/[id].js [hunk 5]
```diff
@@ -0,0 +1,34 @@
...
+    await fetchApi(`/api/setSavedDatas/${savedInsulinData.id}`, {
+      method: 'PUT',
+      headers: {
+        'Content-Type': 'application/json',
+      },
+
+      body: JSON.stringify(updatedSavedInsulinData),
+    });
+
+    console.log('updated!');
```
Comment (K-Juno) [inline pages/setSavedDatas/[id].js]: Here's a console.log!

**Review by K-Juno — CHANGES_REQUESTED** [review]
Just a few typos! Other than that, huge project, good job!

**Comment by FreytagDom** [comment]
Is changed

**Pull request merged**